recipes/ghc/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+export LDFLAGS="-L${PREFIX}/lib"
+export CPPFLAGS="-I${PREFIX}/include"
+ls
```

Comment:
May need to download an existing `ghc` binary to get the build to run through. That or possibly build an older copy of `ghc`, which can use a C compiler, first. A good example is [`go`]( http://github.com/conda-forge/go-feedstock ). Where @nehaljwani did the latter.

Reply:
I will try with the CENTOS6 bindist